.npmrc.sample

```diff
@@ -0,0 +1 @@
+git-tag-version=false
```

Comment:
Actually, we don't need this, we want to use the `npm version <major|minor|patch>` here, as we don't have Jenkins doing the tagging for us

Reply:
Makes sense 🙌, I've just removed that in the last commit